src/main.cpp

```diff
@@ -40,7 +40,11 @@
 
 #include "binding.h"
 
+#ifdef __WINDOWS__
+#include "resource.h"
+#elseif __LINUX__
```

Comment:
@carstene1ns That would be because it [doesn't exist](http://stackoverflow.com/questions/36137376/elseif-vs-elif-c-c-preprocessor) :)